deploy/gk-deploy

```diff
@@ -233,6 +233,7 @@ while [[ $# -ge 1 ]]; do
         ;;
         -abort)
         ABORT=1
+        SKIP_PREREQ=1
```

Comment:
Here different solutions are possible . I'm explicitly handling only when the --abort is used. We could test the both the approaches